src/components/components.scss

```diff
@@ -12,4 +12,6 @@
   'Navbar/Navbar',
   'Sidebar/Sidebar',
   'TextArea/TextArea',
-  'Hero/Hero';
+  'Hero/Hero',
+  'TitleBody/TitleBody';
+  
```

Comment:
What is happening here? Are you missing a new line?

<img width="132" alt="Screen Shot 2021-02-01 at 8 21 21 PM" src="https://user-images.githubusercontent.com/13306885/106551886-0d58e380-64cb-11eb-9c18-2e5ff694f765.png">

Reply:
I'm not sure what the problem is ☹️. I thought there is a new line at 17 as the screenshot shows and I double checked it in the files and there was nothing below it. Might need help on Friday